### ISSUE TITLE
More edition options in the card pack maker

### ISSUE THREAD

**Issue by MeruMSB** [jakelauer/allbadcards#342]
My idea is a card searcher to make easier edit your packs. Like, removing a card who nobody in your games uses or help you know if your card idea was already in the pack. When you have more than 100 pages this becomes tedious to know what you added before.
Disable cards in card packs which you use but aren't made by you would be nice too.

**Comment by dennmuel** [comment]
Hi @MeruMSB , thanks for the suggestions! That would sure be handy (though it sounds like quite a bit of work.) For some of your requests you can use [this little script](https://github.com/dennmuel/custom-packs/blob/master/deck.py) i wrote. It can sort and count your cards and has a duplicate check for newly added cards.
Hope this helps in the meanwhile. :)

**Comment by jakelauer** [comment]
@MeruMSB I agree that some of this sounds reasonable. Namely - it should be possible to do a quick filter of your cards if you want to edit a particular one.
